challenge-03/README.md

```diff
@@ -150,8 +179,27 @@ deve conter no retorno da frase acima é "metro" no lugar de "metros".
 método), que será concatenada com a frase de retorno, mostrando a resposta
 correta, de acordo com os dados inseridos no objeto.
 */
-?
+pessoa.apresentação = function(){
+	
+	var parte1 = 'Olá, eu sou o ' + pessoa.nome + ' ' + pessoa.sobrenome + ',';
+	var parte2 = ' tenho ' + pessoa.idade + ' anos,';
+	var parte3 = 'eu ja caminhei ' + pessoa.caminhouQuantosMetros + ' metros!';
```

Comment:
-1
@MateusJabour legal você ter separado em partes assim, mas dessa forma que você fez, pode ter alguns problemas. O maior delas é a duplicação de código.

Você deve evitar ao máximo duplicar as coisas. Exemplo: na `parte1`, seu eu quisesse mudar a frase de `Olá, eu sou o Fulano de Tal` para `Hey! Meu nome é Fulano de tal`, você teria que lembrar de fazer essa alteração em dois lugares: na variável `parte1`, e também dentro do `if`.

A dica aqui é você colocar em variáveis para validar somente o que for realmente alterar nas frases. Assim você só altera em um lugar, quando for o caso ;)